framework/python/src/common/risk_profile.py

```diff
@@ -152,15 +165,16 @@ def _valid(self, profile_json, profile_format):
 
   def _expired(self):
     # Check expiry
-    created_date = self.created.timestamp()
+    created_date = datetime.strptime(
+      self.created, '%Y-%m-%d').timestamp()
```

Comment:
See note above about keeping this as a timestamp object, no need to decode string when kept as proper timestamp object